XmlSupportPkg/Library/XmlTreeLib/fasterxml/xml_fasterxml.c

```diff
@@ -2700,6 +2700,22 @@ RtlXmlNextToken (
         pToken->fError = TRUE;
         return success;
       }
+      // MS_CHANGE
+      //
+      // TODO-BBB:  See TODO-AAA
+      //
+      // if (pRawToken->TokenName == NTXML_RAWTOKEN_ERROR) {
+      //   pToken->fError = TRUE;
+      //   success = EFI_ABORTED;
+      //   return success;

```

Comment:
Instead of commenting out the code, we can remove it. If it is helpful, do leave a full comment explaining why we don't check if the TokenName == NTXML_RAWTOKEN_ERROR.